lab-kayla/lib/storage.js

```diff
@@ -0,0 +1,66 @@
+'use strict'; //schema = a dragon object
+
+const debug = require('debug')('http:storage');
+const storage = {};
+
+module.exports = exports = {};
+
+exports.createItem = (schema, item) => {
+  debug('#createItem');
+
+  if(!schema) return Promise.reject(new Error('schema required'));
+  if(!item) return Promise.reject(new Error('item required'));
+  if(!storage[schema]) storage[schema] = {};
+
+  storage[schema][item.id] = item;
+
+  return Promise.resolve(item);
+};
+
+exports.fetchItem = (schema, id) => {
+  debug('#fetchItem');
+
+  return new Promise((resolve, reject) => {
+    if(!schema) return reject(new Error('schema required'));
+    if(!id) return reject(new Error('id required'));
+
+    let schemaName = storage[schema]; // = storage.dragon
```

Comment:
**TODO FOR LOST POINTS**

Comments are good. That being said, please manage them on your own branch for your review and submit me a clean branch.